source/openapi3/split.yaml

```diff
@@ -737,13 +715,16 @@ info:
 
     # Sandbox Testing Details
 
-    Try out your happy paths and not-so happy paths, the sandbox is a great place to get started without transferring actual funds. All transactions are simulated and no communication with financial institutions is performed.
+    Try out your happy paths and not-so happy paths, the sandbox is a great place to get started without transferring actual funds. 
```

Comment:
This is a comma splice, so should be either a semicolon (yuck, old) or a dash (sexy, new).

```suggestion
    Try out your happy paths and not-so happy paths — the sandbox is a great place to get started without transferring actual funds. 
```